Purchases/Attribution/RCAttributionTypeFactory.m

```diff
@@ -0,0 +1,55 @@
+//
+// Created by Andrés Boedo on 2/25/21.
+// Copyright (c) 2021 Purchases. All rights reserved.
+//
+
+#import "RCAttributionTypeFactory.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation RCAttributionTypeFactory
```

Comment:
new class, holds everything that we do through "Reflection" i.e.: instantiating classes and methods by name